recipes/NNPOps/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "nnpops" %}
+{% set version = "v0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mikemhenry/NNPOps/archive/refs/tags/v0.01-pre.1.tar.gz
+  #url: https://github.com/openmm/NNPOps/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 87f1af4dc34b00d69fd1c0faee2cf5f23f52b48b93cb8f275415037f2a220573
+
+build:
+  skip: true  # [(not linux) or (cuda_compiler_version in (undefined, 'None'))]
+  number: 0
+  rpaths:
+    - lib/
+    - {{ SP_DIR }}/torch/lib
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64 or ppc64le]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_linux-64  2.17  # [linux64]
+    - make
+    - cmake
+  host:
+    - python 
+    - pytorch 1.10.0
+    - pytorch =*=*cuda*
+    # Skipping test deps since we are running out of diskspace on staged-recipes with testing deps
+    #- torchani
+    #- mdtraj
+    #- pytest
+  run:
+    - python
+    - pytorch 1.10.0
+    - pytorch =*=*cuda*
+    - torchani
+    - mdtraj
+
+test:
+  imports:
+    - NNPOps
+    - NNPOps.BatchedNN
+
+about:
+  home: https://github.com/openmm/NNPOps
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "High performance implementations of operations used in Neural Network Potentials"
+
+extra:
+  recipe-maintainers:
+    - jchodera
+    - mikemhenry
+    - ijpulidos
+    - peastman 
```

Comment:
can you get all these people to agree to maintain? otherwise just add them when it i smerged

Reply:
I can add them back in later, thanks for the suggestion!